docs/packages/Parser.mdx

```diff
@@ -231,7 +231,23 @@ Doing so ensures running ESLint from a directory other than the root will still
 
 > Default `true`.
 
-This option allows you to toggle the warning that the parser will give you if you use a version of TypeScript which is not explicitly supported
+This option allows you to toggle the warning that the parser will give you if you use a version of TypeScript which is not explicitly supported. The warning message would look like this:
+
+```console
```

Comment:
Is `console` a supported language anywhere? I'd think <code>\`\`\`plaintext</code> would be best.

Reply:
good point! was thinking maybe console since this is terminal output but plaintext is more semantically accurate, and is supported by mdx/prismjs. updated this!